public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,95 @@
+# Setup a Google Drive DVC Remote
+
+Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
+[remote storage](/doc/command-reference/remote).
+
+## Create and Configure a "Google Project"
```

Comment:
```suggestion
## Create and Configure a **Google Project**
```
Ignore. Suggestion test.

Reply:
Yep, I was able to commit the suggestion after accepting Max' invitation to his fork. Not a sustainable workflow but at least now we know.